components/markdown/src/markdown.rs

```diff
@@ -516,7 +520,8 @@ pub fn markdown_to_html(
                 let anchor_idx = match context.insert_anchor {
                     InsertAnchor::Left => start_idx + 1,
                     InsertAnchor::Right => end_idx,
-                    InsertAnchor::None => 0, // Not important
+                    InsertAnchor::Heading => 0,  // fixed later
```

Comment:
```suggestion
                    InsertAnchor::Heading => 0,  // modified later to the correct value
```
This looks suspiciously like a FIXME in its current form :)